src/HttpRequest.js

```diff
@@ -6,10 +6,15 @@ const StandardError = require('@unplgtc/standard-error');
 const HttpRequest = {
 	get payload() {
 		return {
-			...this.url != null     ? { url: this.url         } : null,
-			...this.headers != null ? { headers: this.headers } : null,
-			...this.body != null    ? { body: this.body       } : null,
-			...this.json != null    ? { json: this.json       } : null
+			...(this.options != null && this.options),
+			...(this.url != null &&     { url: this.url         }),
+			...(this.headers != null && { headers: this.headers }),
+			...(this.body != null &&    { body: this.body       }),
+			...(this.json != null &&    { json: this.json       }),
+			...(this.qs != null &&      { qs: this.qs           }),
```

Comment:
Should we line up the `&&`s here to match below? I know the bottom one is way too long but it's also a multi-line thing so I think it's a reasonable exception. I'd line up the above `&&`s but only with each other, leave the bottom one as is.

Reply:
Going with lining up `&&`s 👍